src/components/menu-surface/menu-surface.scss

```diff
@@ -11,6 +16,8 @@
 @include menu.core-styles;
 
 .mdc-menu-surface {
+    display: var(--limel-menu-surface-display, block);
+    flex-direction: var(--limel-menu-surface-flex-direction, row);
```

Comment:
something is wrong here 
<img width="724" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/98379a5b-a6fd-4eae-a78a-699a7d1cc7c9">